src/openforms/js/components/admin/form_design/registrations/objectsapi/ObjectTypeSelect.js

```diff
@@ -0,0 +1,60 @@
+import PropTypes from 'prop-types';
+import React, {useEffect} from 'react';
+
+import Select from 'components/admin/forms/Select';
+
+// TODO: properly translate this
+const LOADING_OPTION = [['...', 'loading...']];
```

Comment:
TODO: :)

Reply:
I will resolve them in another PR as I'm using this loading option elsewhere